cmake_build/example_configure.sh.do_not_touch

```diff
@@ -25,8 +25,6 @@ rm -rf CMakeFiles\
 
 cmake ..                                                                            \
 -DCMAKE_C_COMPILER=/usr/bin/gcc                                                     \
--DCMAKE_INSTALL_RPATH="/home/youruser/kratos/libs"                                  \
--DCMAKE_INSTALL_RPATH_USE_LINK_PATH=TRUE                                            \
```

Comment:
Why is this not used any more?
I thought this way one does not have to put Kratos on the LD_LIB_PATH?
fyi @msandre